src/platform/forms/components/review/ApplicationSubmitted.jsx

```diff
@@ -0,0 +1,49 @@
+// libs
+import React from 'react';
+
+// platform - forms components
+import Column from 'platform/forms/components/common/grid/Column';
+import Row from 'platform/forms/components/common/grid/Row';
+
+// platform - forms containers
+import PreSubmitSection from 'platform/forms/containers/review/PreSubmitSection';
+
+// platform - forms-system components
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+
+function ApplicationSubmitted(props) {
+  const { formConfig, goBack, onSubmit } = props;
+
+  return (
+    <>
+      <Row>
+        <Column className="columns" />

```

Comment:
Won't this render `<div classname="columns columns"/>`?

Reply:
not needed here. that is me not properly cleaning up. 😃